stories/Checkbox/CheckboxGroup.stories.jsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import CheckboxGroup from '../../components/Checkbox/CheckboxGroup';
+
+const options = [
+  { label: 'Banana', name: 'banana', key: 'banana' },
+  { label: 'Orange', name: 'orange', checked: true, key: 'orange' },
+  {
+    label: 'Lemon',
+    name: 'lemon',
+    checked: true,
+    disabled: true,
+    key: 'lemon',
+  },
+  {
+    label: 'Strawberry',
+    name: 'strawberry',
+    disabled: true,
+    key: 'strawberry',
+  },
+  { label: 'Blueberry', name: 'blueberry', key: 'blueberry' },
+];
+
+const buttonGroupOptions = [
+  { name: 'with-icon', label: 'With an Icon', icon: 'info' },
+  { name: 'default', label: 'Default' },
+  { name: 'checked', label: 'Checked', checked: true },
+  { label: 'disabled', name: 'disabled', disabled: true },
+];
+
+const getChildren = () =>
+  options.map(({ label, name, checked, disabled }) => (
+    <>
+      <CheckboxGroup.Checkbox
+        label={label}
+        name={name}
+        checked={checked}
+        disabled={disabled}
+        key={name}
+      />
+    </>
```

Comment:
The key warning is still shown in the console. I suggest removing the fragment element or adding a key to it

Reply:
It's more easier understand suggestion when it comes with a description